### PR TITLE
build: Add code coverage and badge

### LINKED ISSUE #122: Code coverage and badge
Add support for code coverage and add a code coverage badge in the README.

### PR DESCRIPTION
**Pull request by stanleyjones**
Closes #122.

### CODE REVIEW

**Comment by stanleyjones** [comment]
Just doing the same thing as https://github.com/liftedinit/gwen/pull/65 but for Alberto.

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,12 +54,16 @@ jobs:
           command: npm run build:<< parameters.app_env >>
       - run:
           name: Compress artifacts
-          command:
-            tar -cvzf /tmp/artifacts/alberto_<< parameters.app_env >>_<< parameters.release_id >>_${CIRCLE_SHA1}.tar.gz ./build
+          command: >
+            tar -cvzf /tmp/artifacts/alberto_<< parameters.app_env >>_<<
+            parameters.release_id >>_${CIRCLE_SHA1}.tar.gz ./build
```
Comment (fmorency) [inline .circleci/config.yml:59]: Can you reformat this line?
```
_<<
parameters.release_id >>_
```
looks wierd.

File: .circleci/config.yml [hunk 2]
```diff
@@ -52,12 +54,16 @@ jobs:
...
       - run:
           name: create release shasum
-          command:
-            shasum /tmp/artifacts/alberto_<< parameters.app_env >>_<< parameters.release_id >>_${CIRCLE_SHA1}.tar.gz > /tmp/artifacts/shasum_<< parameters.app_env >>_<< parameters.release_id >>_${CIRCLE_SHA1}.txt
+          command: >
+            shasum /tmp/artifacts/alberto_<< parameters.app_env >>_<<
+            parameters.release_id >>_${CIRCLE_SHA1}.tar.gz >
+            /tmp/artifacts/shasum_<< parameters.app_env >>_<<
+            parameters.release_id >>_${CIRCLE_SHA1}.txt
```
Comment (fmorency) [inline .circleci/config.yml:66]: Same, fix formatting

**Review by fmorency — APPROVED** [review]
LGTM assuming comments are addressed.

**Pull request merged**